docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -324,16 +324,17 @@ To add a data link, follow these steps:
-   | Method               | Select from **POST**, **PUT**, or **GET**.                                                                                                                |
-   | URL                  | The request URL.</p><p>To add a variable, click in the **URL** field and enter `$` or press Ctrl+Space or Cmd+Space to see a list of available variables. |
-   | Query parameters     | **Key** and **Value** pairs. Click the **+** icon to add as many key/value pairs as you need.                                                             |
-   | Headers              | Comprised of **Key** and **Value** pairs and a **Content-Type**.</p><p>Click the **+** icon to add as many key/value pairs as you need.                   |
-   | Content-Type         | Select from the following: **application/json**, **text/plain**, **application/XML**, and **application/x-www-form-urlencoded**.                          |
-   | Body                 | The body of the request.                                                                                                                                  |
+   | Option               | Description                                                                                                                                                                                                                        |
+   | -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | Title                | A human-readable label for the action that's displayed in the UI.                                                                                                                                                                  |
+   | Confirmation message | A descriptive prompt to confirm or cancel the action.                                                                                                                                                                              |
+   | Method               | Select from **POST**, **PUT**, or **GET**.                                                                                                                                                                                         |
+   | URL                  | The request URL.</p><p>To add a variable, click in the **URL** field and enter `$` or press Ctrl+Space or Cmd+Space to see a list of available variables.                                                                          |
+   | Variables            | **Key**, **Name**, **Type** pairs. Click the **+** icon to add as many variables as you need. To add a variable to the request, prefix the key with `$`. You can configure the values for the variables when performing an action. |
```

Comment:
In this case, "type" shouldn't bolded or capitalized because there's no actual label in the UI. I also made a couple other wording edits.

```suggestion
   | Variables            | **Key** and **Name** pairs with a type selection. Click the **+** icon to add as many variables as you need. To add a variable to the request, prefix the key with `$`. You can set the values for the variables when performing an action. |
```